config.proto

```diff
@@ -21,16 +23,16 @@ message Reporting {
     string address = 1;
 
     // secure when false, permits connecting to the trace endpoint without a certificate
-    bool secure = 2;
+    google.protobuf.BoolValue secure = 2;
 }
 
 // Message describes what message should be considered for certain DataCapture option
 message Message {
     // request describes the outgoing/incoming message in a client/request operation
-    bool request = 1;
+    google.protobuf.BoolValue request = 1;
 
     // response describes the incoming/outgoing message in a client/request operation
-    bool response = 2;
+    google.protobuf.BoolValue response = 2;
```

Comment:
we should use the wrappers for all primitives.

Reply:
agree I have also used it for string.